docs/core/tools/dotnet-install-script.md

```diff
@@ -138,6 +138,18 @@ macOS/Linux:
 
 `./dotnet-install.sh --shared-runtime --version 1.1.0`
 
+#### Windows obtain one-liner example
```

Comment:
I'd add this example following the same format as the others. Intro sentence to indicate what the example is doing and then Windows and macOS/Linux entries. 

Also, it would be good to better describe what this example is doing. Obtain one-liner is not very descriptive. Obtain what?

Reply:
done...